src/bun.js/node/node_fs.zig

```diff
@@ -5217,6 +5220,7 @@ pub const NodeFS = struct {
                 Dirent => {
                     entries.append(.{
                         .name = bun.String.createUTF8(name_to_copy),
+                        .path = bun.String.createUTF8(basename),
```

Comment:
note: this is going to clone the same string value per file in the directory when it doesn't really need to since theyre all the same. `bun.String.createUTF8` creates a WTF::String that can just have a ref-count.